sormas-base/setup/sormas.properties

```diff
@@ -58,19 +58,30 @@ app.url=
 # default: "/opt/sormas/custom/"
 #custom.path=/opt/sormas/custom/
 
+# Key data for the SORMAS to SORMAS feature
+
 # Path where SORMAS to SORMAS related certificates and files are stored
 # default: "/opt/sormas/sormas2sormas/"
-#sormas2sormas.path=/opt/sormas/sormas2sormas/
-
-# Key data for the generated SORMAS to SORMAS certificate"
 #sormas2sormas.path=
+
+# The name of the file containing server access data: organization id, organization name, host name of the server, REST user password
+# default: "/opt/sormas/sormas2sormas/{host name}-server-access-data.csv"
```

Comment:
Technically these are not default values, just the values added by the script, right?
So we probably just write `#sormas2sormas.serverAccessDataFileName=/opt/sormas/sormas2sormas/{host name}-server-access-data.csv`